cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -91,4 +91,19 @@ package object cwl extends TypeAliases {
     import wdl4s.cwl.Implicits.enumerationEncoder
     commandLineTool.asJson
   }
+
+  def encodeCwlWorkflow: Workflow => Json = { workflow =>
```

Comment:
go ahead w/ one-arg function, we're delving into nitpickiness and it's not worth it.